app/test/integration_test.go

```diff
@@ -84,6 +84,14 @@ func (s *IntegrationTestSuite) TearDownSuite() {
 	s.network.Cleanup()
 }
 
+// wait a few blocks to clear the txs
+func (s *IntegrationTestSuite) WaitForNBlocks(num int) {
+	for i := 0; i < num; i++ {
```

Comment:
[nit] 

```suggestion
// WaitForBlocks waits for blockCount number of blocks to be added to the chain.
func (s *IntegrationTestSuite) WaitForBlocks(blockCount int) {
	for i := 0; i < blockCount; i++ {
```

Motivation https://tip.golang.org/doc/comment#funcs